pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -371,9 +371,10 @@ type AbstractVertex struct {
 	// ServiceAccountName to apply to the StatefulSet
 	// +optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty" protobuf:"bytes,15,opt,name=serviceAccountName"`
+	// +optional
 	// +patchStrategy=merge
 	// +patchMergeKey=name
-	Volumes []corev1.Volume `json:"volumes,omitempty" protobuf:"bytes,16,rep,name=volumes"`
+	Volumes []corev1.Volume `json:"volumes,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,16,rep,name=volumes"`
```

Comment:
we have to add `patchStrategy` and `patchMergeKey` explicitly for swagger?

Reply:
For open API.